Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/SpatialCoordinateSystemMember.cs

```diff
@@ -13,62 +13,57 @@
 namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
 {
     /// <summary>
-    /// The SpectatorView helper class for managing the connected observer.
+    /// The SpectatorView helper class for managing a participant in the spatial coordinate system
     /// </summary>
-    internal class ConnectedObserver : DisposableBase
+    internal class SpatialCoordinateSystemMember : DisposableBase
```

Comment:
>SpatialCoordinateSystemMember [](start = 19, length = 29)

Nit: Maybe name this "SpatialCoordinateSystemParticipant" or something? It took me a second to parse "member"